bitrise.yml

```diff
@@ -404,6 +404,17 @@ workflows:
             # Add more specific conditions for directories as needed; otherwise see default case
             # Pattern matches (*) on directory names and reacts to subdirectory changes as well
             for app in $APPS_AFFECTED; do
+                # Check for root directory changes
+                if [[ "$app" == "." ]]; then
+                    CHANGED_FILES=$(git diff --name-only $BASE_BRANCH..."$BITRISE_GIT_COMMIT")
+                    echo "Project root changes detected: $CHANGED_FILES"
+                    ROOT_SPECIFIC_FILES=$(echo "$CHANGED_FILES" | { grep -E '^(bootstrap\.sh|content_blocker_update\.sh|Dangerfile|Dangerfile\.swift|Info\.plist|iOSNimbusFeatureUtility\.sh|package\.lock|Package\.json|Package\.resolved|Package\.swift|rust_components_local\.sh|setup_build_tools\.sh|update_version\.sh|\.swiftlint\.yml)$' || true; })
```

Comment:
If only the README or a .md file is changed, what will we run? building the whole project seems to much but we may need to do a small check?